yarn.lock

```diff
@@ -1543,6 +1543,8 @@
   version "1.3.0"
   resolved "https://registry.yarnpkg.com/@types%2fobject-hash/-/object-hash-1.3.0.tgz#b20db2074129f71829d61ff404e618c4ac3d73cf"
   integrity sha1-sg2yB0Ep9xgp1h/0BOYYxKw9c88=
+  dependencies:
```

Comment:
This is likely causing type checking errors. Remove this and the following line.

Reply:
Thanks! I removed the lines!